superset/views/base_api.py

```diff
@@ -312,49 +313,61 @@ def info_headless(self, **kwargs: Any) -> Response:
         """
         Add statsd metrics to builtin FAB _info endpoint
         """
-        duration, response = time_function(super().info_headless, **kwargs)
-        self.send_stats_metrics(response, self.info.__name__, duration)
-        return response
+        ref = f"{self.__class__.__name__}.info"
+        with log_context(ref, ref, log_to_statsd=False):
```

Comment:
@mistercrunch do you mind clarify why we added logs to these headless requests, as well? It seems all they does is sending statsd metrics. It doesn't make sense to me to keep a log in metadata db for when we send a log to statsd...